x-pack/plugins/apm/public/components/app/diagnostics/summary_tab/index.tsx

```diff
@@ -39,3 +56,68 @@ export function DiagnosticsSummary() {
     </EuiFlexGroup>
   );
 }
+
+function CrossClusterSearchCallout() {
+  return (
+    <EuiCallOut title="Cross cluster search not supported" color="warning">
+      The APM index settings is targetting remote clusters. Please note that
+      this is not currently supported by the Diagnostics Tool and functionality
+      will therefore be limited.
+    </EuiCallOut>
+  );
+}
+
+function PrivilegesCallout({
+  diagnosticsBundle,
+}: {
+  diagnosticsBundle: DiagnosticsBundle;
+}) {
+  const missingClusterPrivileges = Object.entries(
```

Comment:
I think this can be returned by the helper. Similarly to this https://github.com/elastic/kibana/pull/161840/files#diff-8856d0480eba966ed9724ac88fed719f6114573a041d859eabd54745badda4d5R43

Reply:
Fixed